src/GitTools.Core/GitTools.Core.NET40/Git/AuthenticationInfo.cs

```diff
@@ -0,0 +1,8 @@
+namespace GitTools.Git
+{
+    public class AuthenticationInfo
```

Comment:
The GitVersion code doesn't yet do token auth. I want to add it though after we migrate over